packages/react-examples/src/react/Facepile/Facepile.AddFace.Example.tsx

```diff
@@ -9,26 +10,36 @@ const overflowButtonProps = {
 
 export const FacepileAddFaceExample: React.FunctionComponent = () => {
   const [numberOfFaces, setNumberOfFaces] = React.useState(0);
+  const [addFaceAnnounced, setAddFaceAnnounced] = React.useState<JSX.Element | undefined>();
 
   const personas = React.useMemo(() => facepilePersonas.slice(0, numberOfFaces), [numberOfFaces]);
 
   const addButtonProps = React.useMemo(
     () => ({
-      ariaLabel: 'Add a new person',
-      onClick: (ev: React.MouseEvent<HTMLButtonElement>) => setNumberOfFaces(numberOfFaces + 1),
+      ariaLabel: 'Add a new person to the Facepile',
+      onClick: (ev: React.MouseEvent<HTMLButtonElement>) => {
+        const currentFaces = numberOfFaces + 1;
+        setNumberOfFaces(currentFaces);
+        setAddFaceAnnounced(
+          <Announced message={`New person added, currently ${currentFaces} in the Facepile.`} aria-live="assertive" />,
+        );
+      },
     }),
     [numberOfFaces],
   );
 
   return (
-    <Facepile
-      personas={personas}
-      maxDisplayablePersonas={5}
-      overflowButtonProps={overflowButtonProps}
-      overflowButtonType={OverflowButtonType.descriptive}
-      showAddButton
-      addButtonProps={addButtonProps}
-      ariaDescription="To move through the items use left and right arrow keys."
-    />
+    <>
+      {addFaceAnnounced}
```

Comment:
Not sure this is the best pattern. Probably better to have the JSX element here and render it conditionally by toggling a boolean on change of state.

Reply:
So essentially having the `<Announced>` rendering if you have at least one person added and using the `numberOfFaces` state instead of `currentFaces` for the message?